DisguiseUnityRenderStream/Runtime/DisguiseFramerateManager.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+
+namespace Disguise.RenderStream
+{
+    public static class DisguiseFramerateManager
+    {
+#if ENABLE_CLUSTER_DISPLAY
+        public const bool Enabled = false;
```

Comment:
Nitpick: for future-proofing, I prefer to define public APIs with properties (this one is readonly).

Reply:
Fixed